.github/workflows/fix-pull-request.yml

```
@@ -133,8 +142,8 @@ jobs:
         INITIAL_ADMIN_PASSWORD: ${{ secrets.INITIAL_ADMIN_PASSWORD }}
         JWTOPTIONS_SECRETKEY: ${{ secrets.JWTOPTIONS_SECRETKEY }}
         JWTOPTIONS_REFRESH_TOKEN_SECRETKEY: ${{ secrets.JWTOPTIONS_REFRESH_TOKEN_SECRETKEY }}
-        DOCKER_TAG_FRONTEND: ${{ needs.stage_deploy.outputs.frontend_tag }}
-        DOCKER_TAG_BACKEND: ${{ needs.stage_deploy.outputs.backend_tag }}
+        DOCKER_TAG_FRONTEND: ${{ steps.frontend_tag.outputs.frontend_tag }}
```

actionlint (1.7.9)

property "frontend_tag" is not defined in object type {}

(expression)